src/content/self-hosted/helm-configuration.mdx

```diff
@@ -155,6 +155,7 @@ The build service. It's used in combination with `okteto build` to build contain
 - `persistence.storageClass`: The storage class of the persistence volume attached to every buildkit pod.
 - `persistence.size`: The size of the persistence volume attached to every buildkit pod. Defaults to `750Gi`.
 - `persistence.cache`: The size (in Mi) of the buildkit cache to store image caches. It should be around 30Gi smaller than `storage.size`. Defaults to 500Gi.
+- `tolerations`: List of tolerations to be added to the buildkit and registry services. This list of tolerations will be applied in conjunction with any tolerations set in [`tolerations.buildPool`](self-hosted/helm-configuration.mdx#tolerations) until deprecation of `tolerations.buildPool`. If not set, the buildkit Pods will inherit the tolerations list set in [`globals.tolerations.okteto`](self-hosted/helm-configuration.mdx#globals).
```

Comment:
This should only apply yo `buildkit`, not to the registry

Reply:
In the chart, the registry-deployment does inherit tolerations from the [`buildkit.tolerations`](https://github.com/okteto/app/blob/05abcb11331d3c3118c3d16bd7769a85deeea019/chart/okteto/templates/registry-deployment.yaml#L77) and was also in the [Custom Tolerations spec](https://www.notion.so/okteto/Custom-Tolerations-in-Chart-61b648a42e7a4d5b89904d6d9823fb2a?pvs=4#e5fc5b24f2a64f5cbbc976601df98232), but let me know if we are no longer treating them as together under the same Build pool, and the `registry` should only inherit from the Okteto tolerations. 

It should be a relatively quick change in the chart and chart tests.

Reply:
PR for code changes to match documentation changes: https://github.com/okteto/app/pull/8073

Reply:
Fixed the documentation to reflect that the registry will no longer inherit tolerations from the `buildkit.tolerations` helm chart value.